Update TSD Tools Assembly (#1593)
elixir-europe/rdmkit: pages/tool_assembly/tsd_assembly.md

## Who can use the TSD data management tools assembly?

> Resources on TSD itself are accessible for international users through [EOSC](https://marketplace.eosc-portal.eu/services/tds) and directly through the [University of Oslo](https://www.uio.no/english/services/it/research/sensitive-data/access/). Researchers in Norway can, in addition, obtain access through the [national e-infrastructure program Sigma2](https://www.sigma2.no/sensitive-data-services).

Reviewer: > (https://marketplace.eosc-portal.eu/services/tds)

EOSC portal no longer available. This link should change somehow...

Reviewer: Made a specific Issue for these links: https://github.com/elixir-europe/rdmkit/issues/1595

## What can you use the TSD data management tools assembly for?

{% include image.html file="TSD_tool_assembly.svg" caption="Figure 1. Norwegian ELIXIR tools assembly for sensitive data - TSD" alt="TSD tool assembly" %}

The Norwegian Tool Assembly for sensitive data offers support with [Data Management Planning](planning) through a [national instance of the Data Stewardship Wizard (DSW)](https://norway.dsw.elixir-europe.org/wizard/) following the guidelines of the major national and European funding bodies. Dedicated references in DSW guide you (user) through national infrastructure, resources, and laws and regulations, and also include the {% tool "tryggve-elsi-checklist" %} for Ethical, Legal and Social Implications. In the future, you will be able to submit storage request forms for [Data Storage](storage) in TSD with defined access permissions through the DSW.

> In addition to storage services offered by TSD, [processing](processing) and [analysis](analysing) of data is performed in a safe environment within TSD. 

Reviewer: ```suggestion
In addition to its storage services, TSD offers safe environments for data [processing](processing) and [analysis](analysing). 
```

As a national user, you can access TSD by identifying yourself using the Norwegian [ID-porten](https://eid.difi.no/en/id-porten) system. International users can get access by contacting [tsd-contact@usit.uio.no](mailto:tsd-contact@usit.uio.no).

> Within TSD, you can access  a Windows or Linux virtual machine (VM) and, upon request, [high performance computing (HPC) resources](https://www.uio.no/english/services/it/research/sensitive-data/help/hpc/index.html) and [backup storage](https://www.uio.no/english/services/it/research/sensitive-data/help/Backup-file-recovery.html). You login using two-factor authentication with [Google Authenticator](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en) and a dedicated username and password. The [login in procedure](https://www.uio.no/english/services/it/research/sensitive-data/help/login.html) is different for Windows and Linux VMs on TSD.

Reviewer: ```suggestion
Within TSD, you can access a Windows or Linux virtual machine (VM) and, upon request, [high-performance computing (HPC) resources](https://www.uio.no/english/services/it/research/sensitive-data/help/hpc/index.html) and [backup storage](https://www.uio.no/english/services/it/research/sensitive-data/help/Backup-file-recovery.html). You log in using two-factor authentication with [Google Authenticator](https://support.google.com/accounts/answer/1066447?co=GENIE.Platform%3DAndroid&hl=en) and a dedicated username and password. The [login procedure](https://www.uio.no/english/services/it/research/sensitive-data/help/login.html) is different for Windows and Linux VMs.
```
